.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -3,13 +3,18 @@ name: Pull request
 about: Tell us about your contribution
 ---
 
-<!--  Thanks for contributing to Contour!
+---
+## Thanks for contributing to Contour!
+
+Please delete this text between the lines (and the lines) before submitting your PR.
 
 Before submitting a pull request, make sure you read about our Contribution Workflow here: https://github.com/projectcontour/contour/blob/main/CONTRIBUTING.md#contribution-workflow
 
 Some notable call outs from our Contribution Workflow:
-
-1. All PR's should have a `Fixes #NNN` or `Updates #NNN` line in the pull request description. Contour operates according to the talk, then code rule. If you plan to submit a pull request for anything more than a typo or obvious bug fix, first you should raise an issue to discuss your proposal, before submitting any code.
-2. All commits to Contour must have a Developer Certificate of Origin (DCO) sign off. More about that here https://github.com/projectcontour/contour/blob/main/CONTRIBUTING.md#dco-sign-off
-
--->
+1. All PR's must have a `Fixes #NNN` or `Updates #NNN` line in the pull request description.
+Contour operates according to the talk, then code rule.
```

Comment:
Seems like this line should be `#2`

Reply:
I've updated to an unordered list instead.